.pre-commit-config.yaml

```diff
@@ -29,13 +29,6 @@ repos:
         entry: bash -c "git archive HEAD --prefix=apache-arrow/ --output=arrow-src.tar && ./dev/release/run-rat.sh arrow-src.tar"
         always_run: true
         pass_filenames: false
-      - id: cmake-format
-        name: CMake Format
-        language: python
-        entry: python run-cmake-format.py
-        types: [cmake]
-        additional_dependencies:
-          - cmake_format==0.5.2
```

Comment:
Can we keep this by using `archery lint ...`?

Reply:
My long term plan is to use `archery lint` from the pre-commit hooks, but archery doesn't accept a list of files to lint while pre-commit-hooks only check the changed files based on the git diff.

Reply:
OK.